packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -23,15 +23,7 @@ import SyncClient from '../../clients/sync.client.js';
 import configService from '../config.service.js';
 import type { LogLevel } from '../../models/Activity.js';
 import type { Connection, NangoConnection } from '../../models/Connection.js';
```

Comment:
This file should probably be renamed in an effort to reduce confusion with the orchestrator package.

Reply:
you are right. Its functions are createSyncs, deleteSync, getSyncStatus, etc... it is basically a sync manager. I am not a huge fan of the work manager but I don't have a better idea so I am gonna use that for now. Feel free to recommend anything else

Reply:
e05fee61